spec/unleash/client_spec.rb

```diff
@@ -628,4 +627,52 @@
       end
     end
   end
+
+  it "should use custom strategies during evaluation" do
```

Comment:
I've removed most of the legacy tests for strategies, most of them were testing that the base strategies were correctly resolved which no longer makes a lot of sense for us